src/components/cid/Cid.stories.js

```diff
@@ -9,6 +9,9 @@ storiesOf('CID', module)
   .add('CID v0', () => (
     <Cid className='db ma2 monospace' value={'QmYPNmahJAvkMTU6tDx5zvhEkoLzEFeTDz6azDCSNqzKkW'} onClick={action('click')} />
   ))
+  .add('CID v0 identicon', () => (
+    <Cid className='db ma2 monospace' value={'QmYPNmahJAvkMTU6tDx5zvhEkoLzEFeTDz6azDCSNqzKkW'} onClick={action('click')} identicon />
```

Comment:
Change the display to `inline-block` so the identicon appears on the same line as the CID.

```suggestion
    <Cid className='dib ma2 monospace' value={'QmYPNmahJAvkMTU6tDx5zvhEkoLzEFeTDz6azDCSNqzKkW'} onClick={action('click')} identicon />
```

Reply:
Cool. Good fix, having been unfamiliar with Tachyons, I wasn't aware of all these handy classes.